packages/lib-classifier/src/components/Classifier/components/Layout/components/DefaultLayout/DefaultLayout.js

```diff
@@ -21,7 +21,8 @@ const ContainerGrid = styled.div`
     grid-template-columns: 45.333fr 25.333fr;
   }
 
-  @media screen and (max-width: 700px) {
+  // this breakpoint matches Grommet's ResponsiveContext component
+  @media screen and (max-width: 768px) {
```

Comment:
```suggestion
  @media screen and (max-width: ${({ theme }) => theme.global.breakpoints.small.value}px) {
```

We can use the small breakpoint from the theme here, instead of hardcoding the value.

The function syntax is a bit clumsy, so there might be a better way to inject the theme properties.

Reply:
Agreed that there's a way to incorporate theme `size` in Layout components. I'm going to leave that to PR's dedicated to the recently opened design-related Github Issues for unifying the breakpoints.